docs/output.md

```diff
@@ -12,133 +12,142 @@ The directories listed below will be created in the results directory after the
 
 </details>
 
-Proteins are downloaded for input type `taxa` from Entrez.
+## Intermediate results
 
-### Prodigal
+The following intermediate results are generated and written to the output directory as well.
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `prodigal/`
-  - `*.gff`: contains proteins predicted by Prodigal in gff format.
-  - `proteins.pred_*.tsv.gz`: contains proteins predicted by Prodigal in tsv format. The columns are `protein_tmp_id` (<contig-id_suffix>) and `protein_sequence`.
+- `db_tables/`
+  - `microbiomes_entities.nucl.tsv`: matches entities to microbiomes. Contains entity_name, microbiome_id and entity_weight for all entities of input types assembly and bins.
+  - `microbiomes_entities.no_weights.tsv`: matches entities to microbiomes. Contains microbiome_id and entity_id for all unique microbiome - entity combinations.
 
 </details>
 
-Proteins are predicted for input type assembly and bins.
-
-### Generate peptides
+### Download proteins
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `db_tables/`
-  - `peptides.tsv.gz`: contains peptide_id and peptide_sequence for all unique peptides.
+- `entrez_data/`
+  - `entities_proteins.entrez.tsv`: matches temporary protein id given by Entrez to entities. Contains protein_tmp_id and entity_name.
+  - `microbiomes_entities.entrez.tsv`: matches entities (taxa) and their weights to microbiomes. Contains microbiome_id, entity_id and entity_weight for unique microbiome - entity combinations downloaded from Entrez.
+  - `proteins.entrez.tsv.gz`: contains protein_tmp_id (protein id given by Entrez) and protein_sequence for all proteins downloaded from Entrez.
+  - `taxa_assemblies.tsv`: matches taxon id to assembly id.
 
 </details>
 
-Peptides are generated for downloaded or predicted proteins.
+Proteins are downloaded for input type `taxa` from Entrez.
 
-### Report stats
+### Prodigal
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `db_tables/`
-  - `stats.txt`: contains statistics: unique protein counts, total peptide counts, unique peptide counts, unique peptides across all conditions.
+- `prodigal/`
+  - `*.gff`: contains proteins predicted by Prodigal in gff format.
+  - `proteins.pred_*.tsv.gz`: contains proteins predicted by Prodigal in tsv format. The columns are `protein_tmp_id` (<contig-id_suffix>) and `protein_sequence`.
 
 </details>
 
-Some statistics on protein and peptide number are calculated.
+Proteins are predicted for input type assembly and bins.
 
 ### Epitope prediction
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `db_tables/`
-  - `predictions.tsv.gz`: contains peptide_id, prediction_score (epitope prediction score) and allele_id for all unique peptide - allele combinations.
 - `logs/`
   - `prediction_warnings.log`: contains warnings that occured during epitope prediction.
 
 </details>
 
-Epitopes are predicted for unique peptide - allele combinations.
+## Downstream visualizations
```

Comment:
```suggestion
## Downstream visualisations
```

I know that we used British englisch quite often in the outputs, have we decided what to actually use consistent?